style.css

```diff
@@ -408,3 +408,154 @@ form button {
   width: 134px;
   height: 5px;
 }
+
+@media (min-width: 768px) {
+  body {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: "Poppins", sans-serif;
+  }
+
+  .header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 14px 24px;
+  }
+
+  .section {
+    background-position: left top;
+    background-size: cover;
+    justify-content: center;
+    margin: 50px 8px 76px 27px;
+    width: 98%;
+  }
+
+  .title {
+    margin-left: 100px;
+    margin-right: 100px;
+  }
+
+  .info {
+    margin-left: 100px;
+    margin-right: 100px;
+  }
+
+  .works {
+    padding: 114px 142px;
+    gap: 142px;
+  }
+
+  .card1 {
+    display: flex;
+    flex-direction: row;
+    gap: 46px;
+  }
+
+  .card2 {
+    padding: 24px;
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    gap: 46px;
+  }
+
+  .card3 {
+    border-radius: 24px;
+    padding: 24px;
+    display: flex;
+    flex-direction: row;
+    gap: 46px;
+  }
+
+  .card4 {
+    border-radius: 24px;
+    padding: 24px;
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    gap: 46px;
+  }
+
+  .img-container {
+    height: 400px;
+  }
+
+  .button-container {
+    padding: 12px;
+    width: 124px;
+    height: 48px;
+  }
+
+  .button-container3 {
+    background: #4053fc;
+    box-shadow: 1px 8px 16px rgba(64, 83, 252, 0.24);
+    color: #fff;
+    padding: 12px;
+    width: 124px;
+    height: 48px;
+  }
+
+  .about {
+    display: flex;
+    flex-direction: row;
+    padding: 142px;
+    gap: 24px;
+  }
+
+  .container-about {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 24px;
+  }
+
+  .social_media_about {
+    display: flex;
+    flex-direction: column;
+    margin-left: 80px;
+  }
+
+  .social_media_about button {
+    width: 162px;
+    height: 48px;
+  }
+
+  .langList {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px;
+    gap: 12px;
+  }
```

Comment:
- [ ] The elements inside your "langList" are not correctly styled for the desktop version and almost don't appear on the screen. They also should be a card with an image of the language and a text representing that language so that they can be styled differently for the mobile and desktop view. You used an image that is from the mobile version in the desktop version so they are not correctly aligned. Please follow the description of the card I mentioned and create a style for they in the desktop version.
 
 - Your design:
 - 
<img width="1270" alt="image" src="https://user-images.githubusercontent.com/61919617/199732775-fa595a70-fc04-41ac-aada-e3e3718d3480.png">

 - Figma design:
 - 
<img width="855" alt="image" src="https://user-images.githubusercontent.com/61919617/199732915-7ba01237-62d2-4547-b838-968811535a07.png">